code/modules/clothing/suits/labcoat.dm

```diff
@@ -49,3 +49,17 @@
 	name = "scientist labcoat"
 	desc = "A suit that protects against minor chemical spills. Has a purple stripe on the shoulder."
 	icon_state = "labcoat_tox"
+	
+/obj/item/clothing/suit/toggle/labcoat/raincoat
+	name = "Cybersun raincoat"
+	desc = {"
+    A translucent, uniquely designed labcoat worn by the designers of Cybersun Solutions. It's made from a special material that actively repels fluids.   
+    
+    You're pretty sure this is just a raincoat.
+    
+    <i>Wearing a raincoat inside is like wearing sunglasses at night. A good Cybersun exec does both.</i>
+    "}
```

Comment:
Skill issue using spaces instead of tabs. (Also the curly brackets increase performance load :videogames:) (It's probably best to make the description one sentence anyways aswell...)

The thing about it being a special material seems more of a resistance/clothing flag comment (maybe add a flag for repelling liquids being thrown at you from beakers?).

The comment about how it feels to wear sounds like more of a mood thing if anything and IMO subjective comments should not be in item descriptions (What if the character has a different opinion IC? Maybe put it as a slogan on an ad  
 poster or something instead).